image-filters/image-filters.markdown

```diff
@@ -209,15 +205,15 @@ We don’t want our app to be a tiny grey window, so the two essential methods
 that we will start by overriding (implementing in our own class, instead of
```

Comment:
Is it necessary to define "overriding" for this audience? This is maybe a question for @MichaelDiBernardo